dts/bindings/mtd/eeprom.yaml

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2019 Vestas Wind Systems A/S
```

Comment:
I would name this `eeprom-base.yaml`.  Personal preference.

Reply:
@pabigot Thank you. I have renamed the file.